src/api/api-cryptoqr.ts

```diff
@@ -1,17 +1,17 @@
-import { InvoiceApi, type ApiConfig } from './swagger/api'
-import type { Currency } from '@/types/Currency'
+import { InvoiceApi, type ApiConfig } from './cryptoqr/api'
+import type { Currency } from '../types/Currency'
 
-const host = import.meta.env.VITE_HOST
-const basePath = import.meta.env.VITE_BASE_PATH
-const apiKey = import.meta.env.VITE_API_KEY
+const host = import.meta.env.HOST
+const basePath = '/api/v2'
+const merchantCode = 'test'
 
 export default class Api {
 
   private fetchWithMerchantCode = async (url: URL | RequestInfo, init?: RequestInit | undefined): Promise<Response> => {
     init = init || {}
     init.headers = {
       ...init.headers,
-      'x-merchant-code': apiKey,
+      'x-merchant-code': merchantCode,
```

Comment:
hard-codes the merchant code...

Reply:
No change, only a variable name change. I will fix the hard-code later.

Reply:
I just wanted to get rid of the environment input first as that makes no sense

Reply:
Melissa addressed this in her PR